### PR TITLE
[SDKS-1037] SPM integration documentations

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,26 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
 
 ## Installation
 
+### Cocoapods
+Use following pods in your `Podfile` to install FR iOS SDK module(s) you want to install the latest version.
```
Comment (cerebrl) [inline README.md:41]: Missing article: "Use the following pods ... "

Comment (cerebrl) [inline README.md:41]: Is there a missing "if" in the second half of the sentence:

" ... to install FR iOS SDK module(s) if you want to install the latest version."

File: README.md [hunk 2]
```diff
@@ -37,13 +37,26 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
...
 pod 'FRAuth' // Authentication module for Access Manager
 pod 'FRUI' // UI SDK demonstrates FRAuth SDK functionalities
 pod 'FRProximity' // Add-on for FRAuth's Device Profile feature related to location, and BLE
 pod 'FRAutheneticator' // Authenticator module that generates HOTP, TOTP, and Push registration/authentication
+pod 'FRGoogleSignIn' // Social Login module for Google Sign In
+pod 'FRFacebookSignIn' // Social Login module for Facebook Login
 ```
 
+### Swift Package Manager
+> The ForgeRock iOS SDK is only available via Swift Package Manager for 3.0.0 and above. Any versions 2.2.0 and below are only available via Cocoapods
```
Comment (cerebrl) [inline README.md:53]: Missing period at the end of this statement.

File: README.md [hunk 3]
```diff
@@ -37,13 +37,26 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
...
+ 
+1. In Xcode menus, `File` -> `Swift Packages` -> `Add Package Dependencies...`
+2. Select the projec to integrate the ForgeRock iOS SDK
```
Comment (cerebrl) [inline README.md:56]: Typo: "Select the project to ... "

File: README.md (hunk 1 above)
Comment (george-bafaloukas-forgerock) [reply README.md:41]: Maybe even rephrase to "Use the following pods in your `Podfile` in order to install FR iOS SDK module(s)' latest version."

File: README.md (hunk 2 above)
Comment (george-bafaloukas-forgerock) [inline README.md:53]: I would rephrase that to something like:
The ForgeRock iOS SDK is also available via the Swift Package Manager from version 3.0.0 and above. 
The 'FRGoogleSignIn' module is not available via the Swift Package Manager.
Any older versions (2.0.0 and below) are only available via Cocoapods.

**Review by george-bafaloukas-forgerock — CHANGES_REQUESTED** [review]

**Comment by george-bafaloukas-forgerock** [comment]
Some minor changes needed as Justin suggested. I think we need to mention that FRGoogleSignIn is not available through the SPM here as well

File: README.md (hunk 2 above)
Comment (jhoongo) [reply README.md]: I've updated the text blocks adding more information. Please let me know if you have any further concerns.

**Review by spetrov — APPROVED** [review]
Looks good to me! 👍🏻

**Review by cerebrl — APPROVED** [review]

**Review by george-bafaloukas-forgerock — APPROVED** [review]
